cli/args/mod.rs

```diff
@@ -1782,6 +1782,32 @@ impl CliOptions {
       ]);
     }
 
+    // collect unstable granular flags
+    let mut all_valid_unstable_flags: Vec<&str> =
```

Comment:
We could avoid these allocations when `from_config_file.is_empty()`

Reply:
Hello @dsherret ,
Change has been done as you suggested, Checks can be triggered and further review can be done in order to merge changes.